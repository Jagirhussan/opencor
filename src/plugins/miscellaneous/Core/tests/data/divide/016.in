<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <minus/>
            <ci>b</ci>
            <apply>
                <divide/>
                <ci>c</ci>
                <apply>
                    <minus/>
                    <ci>d</ci>
                    <exponentiale/>
                </apply>
            </apply>
        </apply>
    </apply>
</math>
