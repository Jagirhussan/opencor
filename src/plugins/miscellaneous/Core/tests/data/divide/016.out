<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mi mathvariant="italic">a</mi>
        <mo>=</mo>
        <mrow>
            <mi mathvariant="italic">b</mi>
            <mo>−</mo>
            <mfrac>
                <mi mathvariant="italic">c</mi>
                <mrow>
                    <mi mathvariant="italic">d</mi>
                    <mo>−</mo>
                    <mi>e</mi>
                </mrow>
            </mfrac>
        </mrow>
    </mrow>
</math>
